src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -269,6 +269,12 @@ def update_site_configs(resource_group_name, name, slot=None,
                         use32_bit_worker_process=None,  # pylint: disable=unused-argument
                         app_command_line=None):  # pylint: disable=unused-argument
     configs = get_site_configs(resource_group_name, name, slot)
+    if linux_fx_version:
+        if linux_fx_version.strip().lower().startswith('docker|'):
+            update_app_settings(resource_group_name, name, ["WEBSITES_ENABLE_APP_SERVICE_STORAGE=false"])
+        else:
+            delete_app_settings(resource_group_name, name, ["WEBSITES_ENABLE_APP_SERVICE_STORAGE"])
```

Comment:
2 question
1. Is the default value `true` or `false`?
2. what is the consequence when using a custom image with this flag at `false`?